src/CSET/operators/constraints.py

```diff
@@ -62,6 +62,32 @@ def generate_var_constraint(varname: str, **kwargs) -> iris.Constraint:
     return varname_constraint
 
 
+def generate_model_level_constraint(
+    model_level_number: str, **kwargs
```

Comment:
```suggestion
    model_level_number: int | str, **kwargs
```
This should be marked that it can take an integer.